app/src/main/java/org/oppia/android/app/home/HomeFragmentPresenter.kt

```diff
@@ -96,6 +101,7 @@ class HomeFragmentPresenter @Inject constructor(
         when (viewModel) {
           is WelcomeViewModel -> ViewType.WELCOME_MESSAGE
           is PromotedStoryListViewModel -> ViewType.PROMOTED_STORY_LIST
+          is ComingSoonTopicListViewModel -> ViewType.COMING_SOON_TOPIC_LIST
```

Comment:
Is this something we have support from the backend? I'm not so sure about that.

Reply:
Yes we need to introduce Upcoming topics, that will be displayed on home screen when learner has finished all the topics.